plugins/flatpak/gs-flatpak.c

```diff
@@ -2477,18 +2477,57 @@ gs_flatpak_app_remove_source (GsFlatpak *self,
 	return TRUE;
 }
 
+static GsApp *
+gs_flatpak_get_services_app_for_runtime (GsFlatpak *self, GsApp *runtime)
+{
+	const gchar *runtime_id, *runtime_branch, *services_branch, *arch;
+	g_autofree gchar *services_id = NULL;
+	g_autofree gchar *description = NULL;
+	g_autoptr(FlatpakRef) services_ref = NULL;
+	g_autoptr(GError) error = NULL;
+
+	runtime_id = gs_app_get_id (runtime);
+	runtime_branch = gs_app_get_branch (runtime);
+
+	if ((g_strcmp0 (runtime_id, "com.endlessm.Platform") == 0 &&
+	     g_strcmp0 (runtime_branch, "eos3.1") == 0) ||
```

Comment:
Does `com.endlessm.Platform` version `eos3.2` not require `EknServices` version `eos3`?

Reply:
No, there are no knowledge apps that use `com.endlessm.Platform//eos3.2` and never will be.

Reply:
Sorry, did not see the previous similar comment from @joaquimrocha. Sounds good then.